fargate/rehydrate/rehydrator.go

```diff
@@ -18,19 +19,33 @@ func NewRehydrator(s3 *s3.Client) ObjectProcessor {
 }
 
 func (r *Rehydrator) Copy(ctx context.Context, src Source, dest Destination) error {
-	// TODO: file is less than or equal to 100MB ? simple copy : multiPart copy
+	// file is less than 100MB ? simple copy : multiPart copy
+	thresholdSize := 100 * 1024 * 1024 // 100 MB
 	log.Printf("copying %s (size: %v) to bucket: %s key: %s from %s ; ",
 		src.GetName(), src.GetSize(), dest.GetBucket(), dest.GetKey(), src.GetVersionedUri())
 
-	params := s3.CopyObjectInput{
-		Bucket:     aws.String(dest.GetBucket()),
-		CopySource: aws.String(src.GetVersionedUri()),
-		Key:        aws.String(dest.GetKey()),
-	}
-
-	_, err := r.S3.CopyObject(ctx, &params)
-	if err != nil {
-		return err
+	if src.GetSize() < int64(thresholdSize) {
```

Comment:
it's just a little thing, and probably not the most costly of instructions, but if this is iterated 10,000 times, then we coerce `thresholdSize` from int32 to in64 10,000 times. maybe cast to int64 on line 23 above?

Reply:
> it's just a little thing, and probably not the most costly of instructions, but if this is iterated 10,000 times, then we coerce `thresholdSize` from int32 to in64 10,000 times. maybe cast to int64 on line 23 above?

thanks refactored: https://github.com/Pennsieve/rehydration-service/pull/13/commits/37eb352973a96cb8e90e326cf6435fcde83ab471